drms/tests/test_to_datetime.py

```diff
@@ -52,6 +58,20 @@ def test_force_string(time_string, expected):
     assert drms.to_datetime(time_string, force=True) == expected
 
 
-# test_xx_list
-# test_xx_ndarray
-# test_xx_pandas_series
+@pytest.mark.parametrize('time_series, expected', [
+    (data_tai_in, data_tai_out),
+    (pd.Series(data_tai_in), data_tai_out),
+    (tuple(data_tai_in), data_tai_out),
+    (np.array(data_tai_in), np.array(data_tai_out)),
+    (dict(zip(tuple(range(len(data_tai_in))), tuple(data_tai_in))),
+     dict(zip(tuple(range(len(data_tai_out))), tuple(data_tai_out))))
+    ])
+def test_time_series(time_series, expected):
+    drms.to_datetime(time_series).equals(expected)
+
+
+def test_corner_case():
+    assert type(drms.to_datetime('')) == pd._libs.tslib.NaTType
+    assert type(drms.to_datetime('foo')) == pd._libs.tslib.NaTType
+    assert type(drms.to_datetime([])) == pd.Series
+    assert drms.to_datetime([]).empty
```

Comment:
This function needs to be improved. I was going to add a list of valid timestrings and invalid strings together, which would result in a Series with some NaT values. But I couldn't understand how to write `NaT`. The only thing that I was able to compare it with is its data type.